polkadot-protocol/architecture/polkadot-chain/overview.md

```diff
@@ -1,4 +1,371 @@
+
+However, if you prefer to bypass these tools and deal directly with the raw chain data, Polkadot uses the [SCALE codec](https://github.com/paritytech/parity-scale-codec#parity-scale-codec){target=\_blank} to encode block and transaction data. 
+
+This allows you to implement your own codecs for custom interactions with the chain if needed.
+
+### Runtime Upgrades
+
+Runtime upgrades allow Polkadot to change the logic of the chain without the need for a hard fork. 
+
+!!!note
+    You can find a guide for how to properly perform a runtime upgrade [here](runtime upgrades page).
+
+#### Runtime Versioning
+
+There are a number of fields that are a part of the overall `RuntimeVersion`.
+
+Apart the `runtime_version` there is also the `transaction_version` which denotes how to correctly encode/decode calls for a given runtime (useful for hardware wallets). 
+
+The reason `transaction_version` is separate from `runtime_version` is that it explicitly notes that the call interface is broken/not compatible.
+
+## DOT Token
+
+### Tokens
+
+- Token decimals:
+    - Polkadot (DOT): 10
+    - Kusama (KSM): 12
+- Base unit: "Planck"
+- Balance type: `u128`
+
+### What is DOT?
+
+DOT is the native token of the Polkadot network, much like BTC is for Bitcoin and Ether is for the Ethereum blockchain.
+
+### Redomination
+
+Polkadot conducted a community poll, which ended on **27 July 2020** at block **888,888** to decide whether to redenominate the DOT token. 
+
+The stakeholders chose to redenominate the token, changing the value of 1 DOT from 1e12 plancks to 1e10 plancks. 
+
+Importantly, this did not affect the total number of base units (plancks) in the network, only how a single DOT is represented.
+
+The redenomination became effective 72 hours after transfers were enabled, occurring at block **1,248,328** on 21 August 2020 around 16:50 UTC.
+
+## The Planck Unit
+
+The smallest unit of account balance on Substrate-based blockchains (such as Polkadot and Kusama) is called **Planck**, named after the **Planck Length**, the smallest measurable distance in the physical universe. 
+
+Similar to how BTC's smallest unit is the **Satoshi** and ETH's is the **Wei**, Polkadot’s native token DOT is equal to 1e10 Planck, while Kusama’s native token KSM equals 1e12 Planck.
+
+## What are the uses of DOT?
+
+DOT serves four primary functions within the Polkadot network:
+
+1. **Governance**: It is used to participate in the governance of the network.
+2. **Staking**: DOT is staked to support the network’s operation and security.
+3. **Bonding**: It is bonded to connect new chains to Polkadot as parachains.
+4. **Buying Coretime**: used to purchase coretime in-bulk or on-demand and access the  chain to benefit from Polkadot's security and interoperability
+
+Additionally, DOT can serve as a transferable token. For example, DOT held in the Treasury can be allocated to teams developing projects that benefit the Polkadot ecosystem.
+
+## FAQ
```

Comment:
Im not sure if it makes sense to have the FAQ section

Reply:
section removed